_build/maintainer_guide.rst

```diff
@@ -0,0 +1,243 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and it's managed by the `Symfony Docs team`_. This article explains in detail
+some of those management tasks, so it's only useful for maintainers and not
+regular readers or Symfony developers.
+
+Merging Pull Requests
+---------------------
+
+Technical Requirements
+~~~~~~~~~~~~~~~~~~~~~~
+
+* `Git`_ installed and properly configured.
+* ``gh`` tool installed. This is a proprietary CLI tool which only Symfony team
+  members have access to.
+* Some previous Git experience, specially merging pull requests.
+
+First Setup
+~~~~~~~~~~~
+
+First, fork the <https://github.com/symfony/symfony-docs> using the GitHub web
```

Comment:
Markdown detected :robot: 

This should be ``` `https://github.com/symfony/symfony-docs`_ ```. Btw, I think cloning the repository is one that all new maintainers have done already (there is a 100% chance you contributed PRs before becoming maintainer)

Reply:
It nevertheless, for completeness 👍🏻 Maybe someone else will read it because he is interested

Reply:
Yes ... also, you may have forked it in the past and delete your fork ... so it may be a useful reminder.